### PR TITLE
Delete deprecated code

### PR DESCRIPTION
**Pull request by wannaphong**
- delete thaicheck
- delete deletetone
- delete dict_word_tokenize

### CODE REVIEW

File: tests/test_tokenize.py [hunk 1]
```diff
@@ -104,10 +103,6 @@ def test_word_tokenize(self):
         # https://bugs.python.org/issue29620
```
Comment (bact) [inline tests/test_tokenize.py]: Should delete comments in line 104-106 as well.

File: tests/test_util.py [hunk 2]
```diff
@@ -252,7 +250,6 @@ def test_delete_tone(self):
         # https://bugs.python.org/issue29620
```
Comment (bact) [inline tests/test_util.py]: Should delete comments in line 252-254 as well.

File: tests/test_util.py [hunk 3]
```diff
@@ -326,7 +323,6 @@ def test_is_native_thai(self):
...
```
Comment (bact) [inline tests/test_util.py]: Should delete comments in line 326-328 as well.

**Review by bact — CHANGES_REQUESTED** [review]
Great.
Just remove the comments that related to the functions to be removed as well.

**Comment by wannaphong** [comment]
> 
> 
> Great.
> Just remove the comments that related to the functions to be removed as well.

Done 🌷

**Review by bact — APPROVED** [review]
Nice

**Pull request merged**